python/rikai/video.py

```diff
@@ -0,0 +1,137 @@
+from abc import ABC, abstractmethod
```

Comment:
Apache license. 

And rename this to `python/rikai/types/video.py`?

Reply:
Do you think the sampling classes should go in types or in like `python/rikai/utils.py` ?

Reply:
should we have a `python/rikai/video.py` for video related functionalities?

i'd imagine that we will have

```
rikai.types -> for UDT
rikai.torch -> for torch access
rikai.featurestore (or .dataset) -> for dataset management
rikai.video(.sample/.other function) -> for video specific operations?

```

you know what, what about `rikai.transforms.video`?

Reply:
i put everything in rikai.types.video for now. We can decide where to put these utility functions.